index.bs

```diff
@@ -2411,15 +2410,16 @@ additional information about a device,
 or device identifiers,
 each increase the risk of harming the user's privacy.
 
-One risk is that as more specific information is shared,
+The web app should not be able to distinguish between: the user rejecting
+permission to use a sensor/capability; and that sensor/capability not being present.
```

Comment:
```suggestion
The web app should not be able to distinguish between the user rejecting
permission to use a sensor/capability, and the sensor/capability not being present.
```

Reply:
I feel like **sensor/capability** should be generalized to **device** given the principle.

Reply:
We considered that but what we're really talking about is a capability on a device… e.g. bluetooth or ambient light sensor...

Reply:
Isn't this a larger principle?  Namely where designs that depend on something from a user, where the user might reasonably deny that something, those designs should avoid leaking information about denial such that sites might retaliate in some way.  Designs can help with that by making denial indistinguishable from other reasons  that the something might not be available (like in this case, where the something might not even exist).  If that is not possible, then it might be appropriate to manufacture some base rate of failure.

Reply:
@martinthomson: yes, it does seem like a solid larger principle; it [came up in our breakout](https://github.com/w3ctag/meetings/blob/gh-pages/2024/01-london/minutes.md#add-info-to-device-apis-section-about-privacy), and we actually found that the Geolocation API returns a specific result that the request was denied - looks like we should ask if that could be changed.